Doc/c-api/dict.rst

```diff
@@ -93,10 +93,26 @@ Dictionary Objects
    Return ``0`` on success or ``-1`` on failure.
 
 
+.. c:function:: int PyDict_GetItemRef(PyObject *p, PyObject *key, PyObject **pvalue)
```

Comment:
In other functions I used `obj` and `result` instead of `p` and `pvalue`.

Reply:
It seems like the whole PyDict documentation uses *p* for the first parameter. If you want to change it, please open a separated PR to change all of them.

For pvalue, hum, PyDict_Next() uses *pkey* and *pvalue* parameter names... But Python is not always consistent :-( The header files uses *key* and *value* names. The C implementation again uses *pkey* and *pvalue* names. I checked other functions which have a ``PyObject**``: only a small minority uses a ``p`` prefix to show that it's a pointer to an object. So I renamed the parameter to **result**: thanks, good idea.